spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -1181,9 +1181,11 @@ Replace `<app>` with the name of your application and the value of your annotati
 
 An Entry Point Style affects how application properties are passed to the container to be deployed. Currently, three styles are supported:
 
-* `exec`: (default) Passes all application properties as command line arguments.
-* `shell`: Passes all application properties as environment variables.
-* `boot`: Creates an environment variable called `SPRING_APPLICATION_JSON` that contains a JSON representation of all application properties.
+* `exec`: (default) Passes all application properties and command line arguments in the deployment request as container args. Application properties are transformed into the format of `--key=value`.
+* `shell`: Passes all application properties as environment variables. Command line arguments from the deployment request are not converted into environment variables nor set on the container. Application properties are transformed into an uppercase string and `.` characters are replaced with `_`.
```

Comment:
> Command line arguments from the deployment request are not converted into environment variables nor set on the container.

maybe:

> Command line arguments from the deployment request are not converted into environment variables nor set on the container - they are skipped.

Reply:
theres no skipping logic or anything, theres simply nothing done with them so id suggest not stating 'they are skipped'